src/exception/exception-api.ts

```diff
@@ -128,10 +128,14 @@ class ExceptionController {
 
   @HasFullWriteAccess()
   async deleteEntityException(req: Request, res: Response) {
-    const { programId } = req.params;
-    const { entity } = req.body;
+    const { programId, entity } = req.params;
+    const { submitterDonorIds } = req.body;
 
```

Comment:
Do we have any Bad Request handling elsewhere that could catch null/empty values here?

Reply:
not specific to exceptions. I will add that to this https://app.zenhub.com/workspaces/icgc-argo-platform-dk-production-board-5e542d38415f5034e9fed89d/issues/gh/icgc-argo/argo-clinical/952

Reply:
good call